dev-python/pyaml-env/pyaml-env-1.2.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit pypi distutils-r1
+
+DESCRIPTION="Parse YAML configuration with environment variables in Python"
+HOMEPAGE="https://pypi.org/project/pyaml-env/"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="dev-python/pyyaml[${PYTHON_USEDEP}]"
+
+distutils_enable_tests pytest
```

Comment:
Either there are no tests, or they are not available in the pypi tarball. Either remove this line if there are no test, or switch the SRC_URI to the github tarball:

```
============================================ test session starts =============================================
platform linux -- Python 3.11.7, pytest-7.4.4, pluggy-1.3.0 -- /tmp/portage/dev-python/pyaml-env-1.2.1/work/pyaml_env-1.2.1-python3_11/install/usr/bin/python3.11
cachedir: .pytest_cache
rootdir: /tmp/portage/dev-python/pyaml-env-1.2.1/work/pyaml_env-1.2.1
plugins: pkgcore-0.12.24, anyio-4.2.0
collecting ... collected 0 items

=========================================== no tests ran in 0.01s ============================================
* ERROR: dev-python/pyaml-env-1.2.1::localrepo failed (test phase):
```